lib/index.js

```diff
@@ -244,9 +292,10 @@ api.set = async ({
  *
  * @return a Promise that resolves to the token.
  */
-api.get = async ({actor, account, email, type} = {}) => {
+api.get = async ({actor, account, email, type, tokenId} = {}) => {
```

Comment:
Add tokenId doc.

Reply:
Updated. Will fix jsdoc styles once #31 is merged.